gulpfile.ts

```diff
@@ -54,34 +65,43 @@ gulp.task('styles', () => (
       use: [autoprefixer({ browsers: ['last 2 versions'] })],
       compress: true,
     }))
-    .pipe(gulp.dest('dist'))
+    .pipe(gulp.dest('dist/app'))
 ));
 
 gulp.task('__build', gulp.series(
   'clean',
-  gulp.parallel('cards', 'static', 'styles')
+  'typescript',
+  gulp.parallel('cards', 'static', 'styles'),
 ));
 
-
 // Utility functions -------------------------------------------------------------------------------
 
-function buildCardObject(filename, meta, contents) {
+interface CardMeta {
+  authors: string[];
+  categories: string[];
+  created: string;
+  drugs: string|null; // FIXME: This should be an array
```

Comment:
Also, somewhat related...

I noticed on the `pain-management` card, the string values are quoted in the yaml. That's not necessary. Just an FYI